### ISSUE TITLE
Purpose of WooMinecraft.iml?

### ISSUE THREAD

**Issue by JayWood** [WooMinecraft/WooMinecraft#38]
So, I've officially switched off of Eclipse and to IntelliJ IDEA.  I already use PHPStorm from JetBrains and I'm already familiar with 99% of the UI/Keyboard shortcuts so I figured why not!?

So, that begs the question, what's up with the WooMinecraft.iml file?  What's its purpose? Should it be committed to the repo?

**Comment by TekkitCommando** [comment]
@JayWood It is like the .project of eclipse, makes it grap dependencies from a specific directory, tells intellij what JDK to use, etc.

**Comment by JayWood** [comment]
If we're using maven, is that even needed?

**Comment by TekkitCommando** [comment]
@JayWood No :)

**Comment by JayWood** [comment]
So it would be safe to add it to the .gitignore then :dancers:

**Comment by TekkitCommando** [comment]
Yep :D

**Comment by JayWood** [comment]
Noted, will do, as well as I guess the .classpath and .project?

**Comment by TekkitCommando** [comment]
Yeah, its actually recommended not to add the .project and .classpath :P They aren't necessary to have a functioning project :P a .iml file was handy for me since there was no maven or gradle :)

**Comment by JayWood** [comment]
Ahh, well I'm all about maven now that I know how to use it.  It reminds me of other systems we use at work daily ( like bower ).

**Comment by TekkitCommando** [comment]
@JayWood If you don't mind me asking where did you learn php :P

**Comment by JayWood** [comment]
Self taught actually, full back-story here: http://www.plugish.com/about/ I started by reading a book "PHP For Dummies" and just starting making crappy code, which evolved into where I am now.

Been using it for a little over 7 years I think? Maybe 6? I lost track!

**Comment by TekkitCommando** [comment]
xD Sweet I am self taught in Java and Learning JS and all those frameworks like Node, JQuery, Angular, Agile, etc. for it. Then I think I will work on PHP :)

**Comment by JayWood** [comment]
Yea definitely start with those before PHP ( I still don't know node or angular ), I use jQuery almost daily.  As a company we've talked about standardizing our Javascript and using a framework instead of jQuery, something like React.

I'm of course self-taught in java, as evident by my screwey code when you forked the project.  I've since evolved :smile: and hope to be doing more java code on the weekends.
